src/media/local-stream.ts

```diff
@@ -283,6 +283,32 @@ abstract class _LocalStream extends Stream {
     return this.effects;
   }
 
+  /**
+   * Method to serialize data about input, output streams
+   * and also effects from _LocalStream.
+   *
+   * @returns - A JSON-compatible object representation with data from _LocalStream.
+   */
+  toJSON() {
```

Comment:
nit: let's do `toJson`

Reply:
`toJson` won't work, it has to be `toJSON`, because that's what `JSON.stringify()` calls, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
Oh, sorry didn't realize this was an automatic hook for JSON.stringify. 👍